crates/burn-tensor/src/tensor/api/numeric.rs

```diff
@@ -742,6 +757,47 @@ where
         )
     }
 
+    /// Returns the indices of the `k` largest elements of the given input tensor along a given dimension.
+    ///
+    /// # Arguments
+    ///
+    /// * `k` - Number of top indices to return.
+    /// * `dim` - The dimension along which to find top elements
+    ///
+    /// # Returns
+    ///
+    /// A new tensor containing the indices of the `k` largest elements along a given dimension.
+    ///
+    /// # Example
+    /// ```rust
+    /// use burn_tensor::backend::Backend;
+    /// use burn_tensor::{Int, Tensor};
+    ///
+    /// fn example<B: Backend>() {
+    ///    let device = B::Device::default();
+    ///    let tensor = Tensor::<B, 2, Int>::from_ints(
+    ///        [
+    ///          [1, 2, 3],
+    ///          [6, 5, 4],
+    ///          [7, 9, 8]
+    ///        ],
+    ///        &device
+    ///    );
+    ///
+    ///    let tensor = tensor.argtopk(2, 1);
+    ///    println!("{}", tensor);
+    ///    // Tensor { data: [
+    ///    //   [2, 1],
+    ///    //   [0, 1],
+    ///    //   [1, 2]
+    ///    // ], ... }
+    /// }
+    /// ```
+    pub fn argtopk(self, k: usize, dim: usize) -> Tensor<B, D, Int> {
```

Comment:
This can be removed from the PR (see earlier comments).

Reply:
see first comment